WordPress/src/main/res/values-small/dimens.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="avatar_sz_login">64dp</dimen>
+    <dimen name="avatar_sz_login_epilogue">38dp</dimen>
+    <dimen name="margin_extra_extra_large">24dp</dimen>
+    <dimen name="margin_extra_large">8dp</dimen>
```

Comment:
`margin_extra_extra_large` and `margin_extra_large` are used in places beyond the login screens so, changing them will affect a great number of screens. Let's try to minimize the scope and introduce new definitions that are only used in Login instead.

To try to keep it a smaller change, let's only introduce new ones in bottom/top margins to fit the small screen heights.